source/features/jump-to-conversation-close-event.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '.';
+
+function init(): Deinit {
+	return observe('#partial-discussion-header :is([title="Status: Closed"], [title="Status: Merged"]):not(.rgh-jump-to-conversation-close-event)', {
+		constructor: HTMLSpanElement,
+		add(messageContainer) {
+			messageContainer.classList.add('rgh-jump-to-conversation-close-event');
+			// Hide the native title
```

Comment:
@fregante is this a good idea? We cannot modify the title since that is out detection, but I dont want the title tooltip to show.

Reply:
I agree that it's not particularly useful, but let's not mess with it

Reply:
It gets in the way of the other tooltip

Reply:
Can you extend the comment explaining why we need to leave the title attribute, why we can’t modify it, and why it needs to be hidden at all?

The current comment only explain what it’s doing but not why